Makefile

```
@@ -5,6 +5,10 @@ GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
 ## Tool Versions
 GOLANGCI_LINT_VERSION ?= v2.8.0
 
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD 2>/dev/null || echo unknown)
+BUILD_DATE ?= $(shell date -u +%Y-%m-%dT%H:%M:%SZ)
+LDFLAGS ?= -X main.commit=$(GIT_COMMIT) -X main.date=$(BUILD_DATE)
```

checkmake (0.3.2)

[warning] Variable "BUILD_DATE" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)